mkosi

```diff
@@ -946,6 +946,10 @@ def mount_image(args: CommandLineArguments,
         yield
     finally:
         with complete_step('Unmounting image'):
+
+            for d in ("home", "srv", "efi", "run", "tmp"):
```

Comment:
not a fan of keeping this list growing. Particularly as there are plans to support arbitrary partitions taken from `mkosi.sfdisk`.